demisto_sdk/commands/common/constants.py

```diff
@@ -939,6 +934,22 @@ class PB_Status:
     }
 ]
 
+FEED_OPTIONAL_PARAMS_SERVER_IMP = [
+    {
+        'additionalinfo': "Incremental feeds pull only new or modified indicators that have been sent from the"
+                          "integration. As the determination if the indicator is new or modified happens on the"
+                          " 3rd-party vendor's side, and only indicators that are new or modified are sent to"
+                          " Cortex XSOAR, all indicators coming from these feeds are labeled new or modified.",
+        'display': 'Incremental Feed',
+        'name': 'feedIncremental',
+        'options': ['RED', 'AMBER', 'GREEN', 'WHITE'],
```

Comment:
```suggestion
```

Reply:
This causes validate failure, the options are checked and should be kept